### PR TITLE
Awesome-Books

### PR DESCRIPTION
**Pull request by Donmark2k**
### Highlights 🥇

**Awesome books: refactor to use JavaScript classes`**

**Learning objectives**

- Use JavaScript classes.
- Understand how to use medium-fidelity wireframes to create a UI.

**What I Have Done:**

- Added linters. The [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
- Create a collection that keeps a list of books.
- Create a function to add a new book to the collection.
- Display all books saved in the collection in the top part of the page
- When a user clicks the "Add" button, a new book is added to the collection
- When a user clicks the "Remove" button,the correct book is removed from the collection.
- The data is stored in the browser's memory using localStorage
- Implement only a basic UI with plain HTML
- Use JavaScript classes instead of objects and arrays.
- I used objects and arrays inside classes.
- Create class methods to add and remove books

**Followed Rules:**

- Made sure [no linter errors](https://github.com/microverseinc/linters-config)
- Followed [Github Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
- Documented [professionaly](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
- Followed the list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- Followed our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

### CODE REVIEW

**Review by HAMMAS-SALEEM — APPROVED** [review]
Great work on implementing some requirements 👍

File: README.md [hunk 1]
```diff
@@ -254,7 +256,7 @@ If you like this project...
```
Comment (HAMMAS-SALEEM) [inline README.md:273]: Capture

- [ ] You didn't fill this section of your readme 😨 Kindly add the instructions here of how to start your application. 👍

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Donmark2k, @Victor-chiemerie :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**